text/0013-code-formatters.md

```diff
@@ -0,0 +1,27 @@
+# Code formatters
+- Start Date: Mid 2016
+- RFC PR: https://github.com/CanopyTax/frontend-rfc/pull/13
+
+## Squad vs Chapter vs Frontend-wide
+Each squad may choose whether to use a code formatter and, if so, which one to use.
+
+## Definition
+A code formatter is a tool that changes source code to have consistent indentation, styling, and formatting.
```

Comment:
Should the definition specify a difference between formatting and linting? Not all lint rules are about formatting or style. Also, we have Canopy's eslint config. Is that standard?

Reply:
👍 yeah i think explaining the connection between code formatting and linting would be good. I've been planning on having a separate rfc for linting. Do you think it makes sense to separate them? Or should code formatters and styling/linting be in the same rfc?

Reply:
🤷‍♂️ Often times they are associated with one another

Reply:
seems that may be "related, but different", where this is **not** talking about , erm, coding best practice and syntactical linting

Reply:
My vote is that I follow up this PR with one for Styling/Linting and then we link the two.

Or maybe if it turns out you can't talk about one without the other, we combine them